express/scripts/martech.js

```diff
@@ -357,10 +357,10 @@ loadScript('https://www.adobe.com/marketingtech/main.min.js', () => {
 
     // quick actions clicks
     } else if ($a.href.includes('spark.adobe.com/tools')) {
+      // nothing yet
     } else if ($a.href.match(/spark\.adobe\.com\/[a-zA-Z-]*\/?tools/g)) {
       adobeEventName = appendLinkText(adobeEventName, $a);
       sparkEventName = 'quickAction:ctaPressed';
-      
```

Comment:
these 2 lines produced errors in `npm run lint` so either somebody ignored them or lint was added later